frontend/src/components/Dialog/types.ts

```diff
@@ -0,0 +1,10 @@
+export default interface DialogProps {
+  title: string;
+  content: string;
+  openDialogState: boolean;
+  closeFunction: () => void;
+  createFunction: (val: string) => void;
+  closeBtnText: string;
+  doneBtnText: string;
+  placeHolderText: string;
+}
```

Comment:
@yishain11 can be moved inside `Dialog.tsx` (10 lines are bearable)

Reply:
@yishain11
regarding the updated file:
-  it's now a 19 lines file, which is still bearable if lived inside `Dialog.tsx`
- fix typos `cancleBtnText` and `cancleFunction`

Reply:
done